tock/tock/remote_user_auth.py

```diff
@@ -43,6 +44,12 @@ def verify_userdata(user):
 
 
 class TockUserBackend(UaaBackend):
+
+    def user_can_authenticate(self, user):
+        if not user.is_active:
+            raise PermissionDenied(ACCOUNT_INACTIVE_MSG)
```

Comment:
This is an interesting way of handling this. In my head, I think I saw either redirecting to a simple templateView or maybe leveraging the messages framework, but I think I like piggybacking on the 403.